<ui version="4.0" >
 <class>ImageSequenceWidget</class>
 <widget class="QWidget" name="ImageSequenceWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="mImageLabel" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="mPlayButton" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset>images/player_play.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="mSlider" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
